PhysicsTools/PatAlgos/python/selectionLayer1/lowPtElectronSelector_cfi.py

```diff
@@ -6,7 +6,7 @@
 #
 selectedPatLowPtElectrons = cms.EDFilter("PATElectronSelector",
     src = cms.InputTag("patLowPtElectrons"),
-    cut = cms.string("pt>1. && electronID('ID')>1.5"),
+    cut = cms.string("pt>1. && electronID('ID')>0.0"),
```

Comment:
is this the new 10% point matching the new training?

Reply:
Yes, although this has been updated again due to the use of a new (smaller) model in cms-data/RecoEgamma-ElectronIdentification#16. The new threshold is -0.25, which again provides a 10% mistag rate and ~90% signal efficiency.